index.html

```diff
@@ -3,9 +3,35 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+    <link rel="stylesheet" href="./fonts/CreteRound-Italic.ttf">
     <title>Mobile Portfolio</title>
 </head>
 <body>
-    <h1>My Portfolio</h1>
+    <header class="top-navigation">
```

Comment:
- The logo has a link (a tag). Kindly consider implementing this for easy navigation